sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -1,607 +1,329 @@
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-      <item>
-          <EFDB_Name>tangentLink60DegAbsolutePositionSteps</EFDB_Name>
-          <Description>Present position in absolute steps at 60 degree, increasing sign indicates extension towards the mirror.</Description>
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-      <item>
-          <EFDB_Name>tangentLink120DegAbsolutePositionSteps</EFDB_Name>
-          <Description>Present position in absolute steps at 120 degree, increasing sign indicates extension towards the mirror.</Description>
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-      <item>
-          <EFDB_Name>tangentLink180DegAbsolutePositionSteps</EFDB_Name>
-          <Description>Present position in absolute steps at 180 degree, increasing sign indicates extension towards the mirror.</Description>
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-      <item>
-          <EFDB_Name>tangentLink240DegAbsolutePositionSteps</EFDB_Name>
-          <Description>Present position in absolute steps at 240 degree, increasing sign indicates extension towards the mirror.</Description>
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-      <item>
-          <EFDB_Name>tangentLink300DegAbsolutePositionSteps</EFDB_Name>
-          <Description>Present position in absolute steps at 300 degree, increasing sign indicates extension towards the mirror.</Description>
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_axialActuatorAbsolutePositionSteps</EFDB_Topic>
```

Comment:
What about we shorten this to MTM2_axialActuatorSteps?
Even if the long topic name doesn't create a problem for EFD, it is not convenient to use.

Reply:
Fixed in commit: 9bb8760. Thanks!